modules/layouts/LayoutCS.chpl

```diff
@@ -0,0 +1,656 @@
+/*
+ * Copyright 2004-2017 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Debug flag */
+config param debugCS = false;
+use RangeChunk only ;
+
+/* For backwards-compatibility with LayoutCSR */
+type CSR = CS;
```

Comment:
Is this sufficient to keep existing `LayoutCSR` programs working?  If so, I think we should rewrite all `use LayoutCSR` tests to `use LayoutCS`, rip out the guts of LayoutCSR as you propose in your PR message, add a deprecation warning to that module, and add a test/deprecated/ test to help us remember to remove it after 1.16 goes out.  This could either be done in this PR or as a quick follow-on.

If it's not sufficient, I'd remove this line.